cvat/apps/dataset_manager/formats/datumaro.py

```diff
@@ -15,29 +14,23 @@
 
 from .registry import dm_env, exporter, importer
 
-class DeleteImagePath(ItemTransform):
-    def transform_item(self, item):
-        image = None
-        if item.has_image and item.image.has_data:
-            image = Image(data=item.image.data, size=item.image.size)
-        return item.wrap(image=image, point_cloud='', related_images=[])
-
 
 @exporter(name="Datumaro", ext="ZIP", version="1.0")
 def _export(dst_file, temp_dir, instance_data, save_images=False):
     dataset = Dataset.from_extractors(GetCVATDataExtractor(
         instance_data=instance_data, include_images=save_images), env=dm_env)
-    if not save_images:
-        dataset.transform(DeleteImagePath)
     dataset.export(temp_dir, 'datumaro', save_images=save_images)
 
     make_zip_archive(temp_dir, dst_file)
 
-@importer(name="Datumaro", ext="ZIP", version="1.0")
+@importer(name="Datumaro", ext="JSON, ZIP", version="1.0")
```

Comment:
It looks like `Datumaro 3D` can also be extended to support importing annotations in JSON format